### ISSUE TITLE
Doesn't seem to process numbers

### ISSUE THREAD

**Issue by majapw** [kesne/jeopardy-bot#185]
See "The Dating Game" category where the answers are all years. :/

**Comment by majapw** [comment]
The answer `W nineteen eighty 1980` also didn't get processed? Like no response, good or bad? Maybe it's an issue if the answer ends with a numerical value.

**Comment by majapw** [comment]
Okay, we have empirically determined that if you add nonsense to the end, ie `w 1945 b` it deals with the number correctly.

**Comment by kesne** [comment]
That's weird. It looks like it should process: https://github.com/kesne/jeopardy-bot/blob/master/src/trebek/commands/guess.js#L10

I'll have to dive into the message processing and see what happened.

**Comment by kesne** [comment]
So I finally figured this out. Numbers with prepended text get interpreted as selecting a clue. The regex in this case that is causing trouble is this:

```
/(?:ill take |give me |choose )?(.*?) (?:for )?\$?(\d{3,4})(?: alex| trebek)?/,
```

Recently, we made the text "for" optional, so this is a side-effect of removing that requirement in clue selection. This causes us to have overlapping regexes when the end of your clue is 3-4 numbers (which makes all year questions impossible to answer). This gives us a couple options:
- Option 1: Revert that change and require “for” again.
- Option 2: Manually exclude the starting words for guessing from clue selection.
